pod/recorder/models.py

```diff
@@ -79,32 +142,56 @@ def process_recording(sender, instance, created, **kwargs):
 
 
 class RecordingFile(models.Model):
-    file = models.FileField(upload_to='uploads/')
-    type = models.CharField(
-        max_length=50, choices=RECORDER_TYPE, default=RECORDER_TYPE[0][0])
+    file = models.FilePathField(path=DEFAULT_RECORDER_PATH, recursive=True,
+                                unique=True, help_text=_(
+                                    'Source file of the published video.'))
+    file_size = models.BigIntegerField(_('File size'), default=0)
```

Comment:
Why a file size field ?

Reply:
Used to calculate if the file is fully upload by the recorder at each exec of the command

Reply:
ok